GUI/GUI.py

```diff
@@ -105,3 +104,21 @@ def end_frame(self) -> None:
     def close(self) -> None:
         self.impl.shutdown()
         glfw.terminate()
+
+
+class GUI_helper:
+    """
+    This class provides helper functions for creating GUI elements.
+
+    Methods:
+        add_spacer():
+            Adds a horizontal line with some padding."""
+
+    def __init__(self):
```

Comment:
The init is probably not necessary, you can remove it since it does nothing. There are no member variables to initialize and no parameters to set, nor should this class every be instantiated.